tests/unit_tests/tools/test_compiler_wrapper.py

```diff
@@ -0,0 +1,348 @@
+##############################################################################
+# (c) Crown copyright Met Office. All rights reserved.
+# For further details please refer to the file COPYRIGHT
+# which you should have received as part of this distribution
+##############################################################################
+
+'''Tests the compiler wrapper implementation.
+'''
+
+from pathlib import Path, PosixPath
```

Comment:
I've never found I need to explicitly import `PosixPath`. For instance  `cwd = Path()` should give the currently selected directory.

Reply:
Good point.